ProcessMaker/Http/Controllers/HomeController.php

```diff
@@ -10,12 +10,6 @@ class HomeController extends Controller
 {
     public function index(Request $request)
     {
-
-      if (Auth::check()) {
```

Comment:
This should still exist. It checks if the user is logged in.

Reply:
The auth in the route is already checking so I thought it might be redundant but I can put it back